src/components/Sidebar/Properties/PropertyOwnerHeader.jsx

```diff
@@ -284,13 +291,13 @@ function PropertyOwnerHeader({
         </span>
         <span className={styles.rightButtonContainer}>
           { isSceneObject && focusButton }
-          { hasMoreButtons && (
+          { (showPopOutSettings || showMeta || showMeta) && (
```

Comment:
```suggestion
          { (showPopOutSettings || showMeta) && (
```

Reply:
Ah :) Actually, it should be: 
```suggestion
          { (showPopOutSettings || showMeta || trashAction) && (
```